.jenkins/job_beam_PreCommit_Website_Stage.groovy

```diff
@@ -0,0 +1,66 @@
+import common_job_properties
```

Comment:
Yeah I'd thought of that but I'm pretty resistant to having the jobs be dependent on the configuration of their environment to succeed -- if something goes wrong with the clone it could adversely affect the jobs running in Jenkins. Having all of our jenkins jobs defined in one place seems like a better choice.